src/projectservice/index.js

```diff
@@ -70,9 +72,63 @@ app.post('/createProject', async (req, res) => {
   })
 })
 
+// Endpoint for deleting a project
+app.delete('/deleteProject/:projectID', async (req, res) => {
+  let token = req.body.token
+  if (!token) {
+    const { username, password } = req.body
+    token = await getToken(username, password)
+  }
+  const projectID = req.params.projectID
+  try {
+    const status = await deleteProject(projectID, token)
+    const data = {
+      // eslint-disable-next-line quote-props
+      'status': status.message,
+      // eslint-disable-next-line quote-props
+      'TaigaAPIResponseStatus': status.success
+    }
+    if (status.success) {
+      res.status(201).send(data)
+    } else {
+      res.status(500).send({
+        message: 'Project ID is invalid'
+      })
+    }
+  } catch (error) {
+    res.status(500).json({ error: error.message })
+  }
+})
+
+// Endpoint for Updating a Project
+app.patch('/updateProject/:projectID', async (req, res) => {
+  let token = req.body.token
+  if (!token) {
+    const { username, password } = req.body
+    token = await getToken(username, password)
+  }
+  const projectID = req.params.projectID
+  const patch = req.body.patch
+  try {
+    const projectData = await editProject(token, projectID, patch)
+    // eslint-disable-next-line eqeqeq
+    if (projectData.status == 'success') {
+      res.status(201).json(projectData.data)
+    } else {
+      res.status(500).json({
+        message: 'Error in editing project name'
+      })
+    }
+  } catch (error) {
+    res.status(500).json({ error: 'Error editing project' })
+  }
+})
+
 // Start the server
 app.listen(port, () => {
-  console.log(`Project microservice running at http://localhost:${port}`)
+  console.log(
+    `Create Project microservice running at http://localhost:${port}`
+  )
 })
 
```

Comment:
Its not create project microservice, it is project microservice, switch back the former name